examples/rust-examples/satchel-demo/tests/satchel_demo.rs

```diff
@@ -93,8 +32,43 @@ pub mod tests {
     }
 
     #[test]
-    fn it_fails_intentionally() {
-        assert_eq!(multiply(-1, -1), -2, "This is an intentional failure!");
+    #[should_panic]
+    fn test_divide_by_zero_panics() {
+        fn get_zero() -> i32 {
+            0
+        }
+        let _result = 1 / get_zero();
+    }
+
+    #[test]
+    #[should_panic(expected = "attempt to divide by zero")]
+    fn test_divide_by_zero_with_message() {
+        fn get_zero() -> i32 {
+            0
+        }
+        let _result = 1 / get_zero();
+    }
+
+    // using format! in the panic message
+    #[test]
+    #[should_panic(expected = "invalid multiplier: 42")]
+    fn test_custom_panic_formatted() {
+        let invalid_value = 42;
+        panic!("invalid multiplier: {}", invalid_value);
+    }
+
+    // verifies only part of a longer message
+    #[test]
+    #[should_panic(expected = "multiplier")]
+    fn test_custom_panic_partial_match() {
+        panic!("Error: invalid multiplier in calculation");
+    }
+
+    // custom panic message
+    #[test]
+    #[should_panic(expected = "invalid value")]
```

Comment:
libtest supports three different ways of specifying the argument:

1. `#[should_panic(expected = "...")]`
2. `#[should_panic("...")]`
3. `#[should_panic = "..."]`

From my testing, all three do work with satchel, but please modify these tests so that all three different syntaxes are used.